libs/agora/teams/src/services/team.service.ts

```diff
@@ -0,0 +1,64 @@
+// -------------------------------------------------------------------------- //
+// External
+// -------------------------------------------------------------------------- //
+import { Injectable } from '@angular/core';
+import { Observable, of, map } from 'rxjs';
+import { tap, share, finalize } from 'rxjs/operators';
+
+// -------------------------------------------------------------------------- //
+// Internal
+// -------------------------------------------------------------------------- //
+import { Team, TeamsResponse } from '@sagebionetworks/agora/models';
+import { ApiService } from '@sagebionetworks/agora/services';
+
+// -------------------------------------------------------------------------- //
+// Service
+// -------------------------------------------------------------------------- //
+@Injectable()
+export class TeamService {
+  teams: Team[] = [] as Team[];
+  teamsObservable: Observable<TeamsResponse> | undefined;
```

Comment:
Would it be OK for Agora to adopt this naming convention for Observable variables: `teams$`? This is widely adopted in the TS community and help decluter the code.

Reply:
Absolutely.  Some of the code that is being migrated might have preceded these conventions.  Thanks for pointing this out.  I will look out for these observables as I am migrating.